index.html

```diff
@@ -0,0 +1,65 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <nav>
+            <h3 class="logo">My Logo</h3>
+        </nav>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure that the logo has a link (i.e. it should be wrapped in a `<a>` tag). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. :pray: e.g.:
```HTML
<a href="#"> <h3 class="logo">My Logo</h3></a>
```